packages/shared/lib/services/activity/activity.service.ts

```diff
@@ -261,7 +266,7 @@ export async function activityFilter(environment_id: number, filterColumn: 'conn
 
     const distinctValues: string[] = logs
         .map((log: Record<string, string>) => log[filterColumn] as string)
-        .filter((value: string | undefined): value is string => typeof value === 'string');
+        .filter((value: string | undefined): value is string => typeof value === 'string' && value !== '');
```

Comment:
could be done is sql no? `where <> ''`

Reply:
Indeed, updated with https://github.com/NangoHQ/nango/pull/1825/commits/5f9dbd2066bb0472fd69e4d1cf823767083753a8